release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -77,6 +77,14 @@ OCI layouts consist of an `index.json` file that identifies the images held with
 
```

Comment:
And I see that you took it from the above entry, which was incorrect. I know it's not yours, but could you please fix theirs too? You would replace the line `.Supporting multi-arch OCI local images for catalogs with oc-mirror` with:

```
[id="oc-mirror-multi-arch-oci-local-images"]
==== Supporting multi-arch OCI local images for catalogs with oc-mirror
```

Reply:
Thank you